docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst

```diff
@@ -1,240 +0,0 @@
-
-.. code:: ipython3
-
-    v = grid.at_link['veg_atlink']
-    tec.populateGrids(grid,tfc,tau_cr,tau_crv,v)
-    print(grid.at_cell.keys())
-    print(grid.at_node.keys())
-
-
-.. parsed-literal::
-
-    ['tidal_innundation_rate_cell', 'effective_water_depth_cell', 'topographic_elevation_cell', 'hydroperiod_cell', 'water_depth_at_MHW_cell', 'lev_at_cell', 'tau_cr_cell', 'ebb_tide_flow__velocity_cell', 'flood_tide_flow__velocity_cell', 'roughness_cell', 'water_depth_at_cell']
-    ['topographic__elevation', 'vegetation', 'mean_water__depth', 'tidal_innundation_rate', 'effective_water_depth', 'hydroperiod', 'water_depth_at_MHW', 'lev_at_node', 'tau_cr_node', 'ebb_tide_flow__velocity_node', 'flood_tide_flow__velocity_node', 'roughness_node', 'water_depth_at_node']
-
-
-Plot tidal flow
-~~~~~~~~~~~~~~~
-
-.. code:: ipython3
-
-    tec.plot_tidal_flow(grid)
-
-
-.. parsed-literal::
-
-    /Users/ningjiehu/landlab/landlab/plot/imshow.py:267: MatplotlibDeprecationWarning: You are modifying the state of a globally registered colormap. In future versions, you will not be able to modify a registered colormap in-place. To remove this warning, you can make a copy of the colormap first. cmap = copy.copy(mpl.cm.get_cmap("YlGnBu"))
-      cmap.set_bad(color=color_for_closed)
-    /Users/ningjiehu/landlab/landlab/plot/imshow.py:307: MatplotlibDeprecationWarning: The 'norm' parameter to Colorbar has no effect because it is overridden by the mappable; it is deprecated since 3.3 and will be removed two minor releases later.
-      cb = plt.colorbar(norm=norm, shrink=shrink)
-    /Users/ningjiehu/landlab/landlab/plot/imshow.py:267: MatplotlibDeprecationWarning: You are modifying the state of a globally registered colormap. In future versions, you will not be able to modify a registered colormap in-place. To remove this warning, you can make a copy of the colormap first. cmap = copy.copy(mpl.cm.get_cmap("pink"))
-      cmap.set_bad(color=color_for_closed)
-    /Users/ningjiehu/landlab/landlab/plot/imshow.py:267: MatplotlibDeprecationWarning: You are modifying the state of a globally registered colormap. In future versions, you will not be able to modify a registered colormap in-place. To remove this warning, you can make a copy of the colormap first. cmap = copy.copy(mpl.cm.get_cmap("magma"))
-      cmap.set_bad(color=color_for_closed)
-
-
-
-.. image:: output_26_1.png
-
-
-
-.. image:: output_26_2.png
-
-
-
-.. image:: output_26_3.png
-
-
-
-.. image:: output_26_4.png
-
-
-
-.. image:: output_26_5.png
-
-
-Consider erosion
-----------------
-
-Set mud erodability
-~~~~~~~~~~~~~~~~~~~
-
-.. code:: ipython3
-
-    mud_erodability = (10**-5)*24*3600;  # mud erodability kg/m2/day!!!
-
-.. code:: ipython3
-
-    ero = tec.totalsedimenterosion_mudsine(grid, mud_erodability)
-    #ok so for some reason erosion value is SUPER high -- no clue why...need to clear that up!
-
-.. code:: ipython3
-
-    for i in range(2):
```

Comment:
I've got an updated version of this for-loop - maybe I'm looking at an older version?

Reply:
That's the old version. Let me close this one and do the PR again.

Reply:
Yes, TidalFlow2ACO_formatted_V2.ipynb should be the new one

Reply:
or TidalFlow2ACO_formatted_V2.rst (for sphinx)